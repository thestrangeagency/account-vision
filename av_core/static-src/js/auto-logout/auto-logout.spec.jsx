import React from 'react';
import { shallow } from 'enzyme';

import AutoLogout from './auto-logout';

test('it renders', () => {
  const wrapper = shallow(<AutoLogout />);
  expect(wrapper).toMatchSnapshot();
});
