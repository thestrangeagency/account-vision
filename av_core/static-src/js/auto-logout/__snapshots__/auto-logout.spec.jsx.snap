// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  aria-hidden="true"
  className="modal fade"
  role="dialog"
  tabIndex="-1"
>
  <div
    className="modal-dialog"
  >
    <div
      className="modal-content"
    >
      <div
        className="modal-body"
      >
        <h1>
          Are you still there?
        </h1>
        <p>
          You are about to be signed out due to inactivity. Please click below to stay signed in.
        </p>
        <p>
          You will be signed out in 
          60
           seconds.
        </p>
      </div>
      <div
        className="modal-footer justify-content-center"
      >
        <button
          className="btn btn-primary"
          data-dismiss="modal"
          type="submit"
        >
          Stay signed in
        </button>
      </div>
    </div>
  </div>
</div>
`;
